assets/js/vue/components/information-product/InformationProductList.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div class="m-2">
+        <h4 class="text-center">Information Products</h4>
+        <DxDataGrid
+            :data-source="data"
+            :show-borders="true"
+            :allow-column-resizing="true"
+            :row-alternation-enabled="true"
+            :column-auto-width="true"
+        >
+            <DxLoadPanel :enabled="true"/>
+
+            <DxPaging :page-size="10"/>
+            <DxPager
+                :visible="true"
+                :allowed-page-sizes="[10, 20, 100, 'all']"
+                :show-page-size-selector="true"
+                :show-info="true"
+                :show-navigation-buttons="true"
+            />
+
+            <DxSearchPanel
+                :visible="true"
+                :width="240"
+                placeholder="Search..."
+            />
+            <DxHeaderFilter
+              :visible="true"
+              :allow-search="true"
+            />
+
+            <DxColumn
+                type="buttons"
+                :width="60"
+            >
+                <DxButton
+                    hint="View/Edit"
+                    icon="link"
+                    :visible="true"
+                    :disabled="false"
+                    @click="viewClick"
```

Comment:
I take it this is to be implemented on a different ticket?

Reply:
Yes, that function of not done yet, different ticket.